Jbarget/oxymore
# creates erotic stories page

### What changes have you made?
- created erotic stories page 

### Which issue(s) does this PR fix?

Fixes #13 

### Screenshots (if there are design changes)
<img width="1422" alt="Screenshot 2020-10-30 at 18 48 03" src="https://user-images.githubusercontent.com/53219789/97739719-83226d00-1ae0-11eb-857c-81b3fda67a17.png">

<img width="246" alt="Screenshot 2020-10-30 at 18 50 59" src="https://user-images.githubusercontent.com/53219789/97740241-460aaa80-1ae1-11eb-8d7d-0d07c81f2802.png">

<img width="246" alt="Screenshot 2020-10-30 at 20 00 38" src="https://user-images.githubusercontent.com/53219789/97746440-9b978500-1aea-11eb-82b4-e533dcd643bf.png">

### How to test
- Go to http://localhost:3000/projects/eroticstories and check that there are 2 x placeholder images and 2 x placeholder texts
- Comment out lines 91 to 124 and comment in line 125 to view the `EroticStoriesPreview` component
